### PR TITLE
Added back the PublishDepsFilePath property

### LINKED ISSUE #2777: .NET Global Tools publish broken
.NET Global Tools publish uses the `PublishDepsFilePath` property to include the deps file, but it no longer exists (.NET Core 3.0 Preview 1).

This bug was introduced here: https://github.com/dotnet/sdk/commit/1936ebd7307cebd6c5d1d8d699b9844f73b8fb2c#diff-7dc833d02f198e5a9f0ab88e229db18aL520.

I believe that the property shouldn't have been removed.

### PR DESCRIPTION
**Pull request by jp2masa**
Fixes #2777

## Changes
- Added back the PublishDepsFilePath property.

### CODE REVIEW

**Comment by jp2masa** [comment]
@dotnet-bot test this please

**Comment by jp2masa** [comment]
@dotnet-bot help

**Comment by livarcocc** [comment]
@wli3 and @peterhuene

File: src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets [hunk 1]
```diff
@@ -602,9 +602,13 @@ Copyright (c) .NET Foundation. All rights reserved.
                             _ComputeReferenceAssemblies"
           Condition="'$(GenerateDependencyFile)' == 'true' and '$(_UseBuildDependencyFile)' != 'true'">
 
+    <PropertyGroup>
+        <PublishDepsFilePath Condition=" '$(PublishDepsFilePath)' == '' ">$(PublishDir)$(ProjectDepsFileName)</PublishDepsFilePath>
```
Comment (peterhuene) [inline src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets]: Small nit pick: it looks like we generally use a tab width of 2 spaces in the targets files.

**Review by peterhuene — APPROVED** [review]
@jp2masa Thanks very much for finding and contributing a fix for this issue.  It looks good other than the minor whitespace issue I commented above.  I'm approving still since this will fix the issue, but would you mind amending the commit to use two spaces instead of four?  Thank you very much!

**Pull request merged**